src/index.js

```diff
@@ -76,7 +76,7 @@ const containsNodeModules = (resolvedResource) => {
 };
 
 const findDuplicate = (res) => (t) => {
-    return res.includes(t);
+    return res.includes(t + '/');
```

Comment:
What are your thoughts on a small comment description here as well as using `path.sep` to allow for unix/windows OS?

```suggestion
     // prevent partial name matches. I.e. don't match `/button` when resolving `/button-group`
    const duplicateDir = `${duplicate}${path.sep}`;
    return resolvedResource.includes(duplicateDir);
```